framework_tvb/tvb/interfaces/web/controllers/burst/matjax.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+#
+#
+# TheVirtualBrain-Framework Package. This package holds all Data Management, and
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+
```

Comment:
@VinczeRobert I encourage you to mention yourself as the module author, and to include a short sentence describing the current module at the top.

Reply:
I hesitated mentioning myself because I didn't write this code, I just found part of it in the old tvb (I think that code was written by Mihai Andrei some years ago) and I found the multiline_math_directives_to_matjax method here https://programtalk.com/vs2/?source=python/12878/tvb-library/tvb/basic/traits/util.py (I don't know who wrote this). All I did was put them together and extracting the code from _LeftFragmentAdapter's get_input_tree (old TVB)  in configure_matjax_doc. Maybe I could mention Andrei and myself?